eclipse-scout-core/src/form/fields/slider/SliderField.less

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+@slider-field-container-margin: 20px;
+
+.slider-field {
+  & > .field {
+    display: flex;
+
+    & > input {
+      flex: 20%;
+      min-width: 0;
+      text-align: right;
+    }
+
+    & > .slider-value {
```

Comment:
For consistency reasons, I would name the CSS class "value-label" instead of "slider-value". The name does not have to be globally unique, it is only valid within .slider-field.